style.css

```diff
@@ -78,3 +78,381 @@ span {
 .media {
   padding: 10px;
 }
+
+#twitter {
+  margin-top: 15px;
+}
+
+/* .scroll-down{
+  background-color: white;
+  border-style: none;
+} */
+.recent-works {
+  font-family: "inter", sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.scroll-down {
+  margin-left: 140px;
+  text-align: center;
+  border-width: 0.01em;
+  background: white;
+  border-style: solid;
+  border-color: rgb(211, 210, 210);
+  border-radius: 50%;
+}
+
+.indicator {
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.container3 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
```

Comment:
The content of the project is aligned with flexbox, nice one implementation here, however let us modify our margin-right here or to align our projects in center to look the same on the figma.

![port10](https://user-images.githubusercontent.com/55635977/129384291-170e36bc-8063-4f26-9aac-79d0963ddf25.png)
As you see the right and left side are quite different.